pps_mw_training_environment.yml

```diff
@@ -15,3 +15,4 @@ dependencies:
   - pyproj
   - tensorflow=2.17
   - xarray
+  - scipy
```

Comment:
remove if not needed after your changes.

Reply:
removed